Automattic-Tracks-iOS/Crash Logging/SentryEvent+Extensions.swift

```diff
@@ -0,0 +1,33 @@
+import Sentry
+
+extension Event {
+    static func from(
+        error: NSError,
+        level: SentryLevel = .error,
+        user: Sentry.User? = nil,
+        timestamp: Date = Date(),
+        extra: [String: Any] = [:]
+    ) -> Event {
+        let event = Event(level: level)
+
+        if let underlyingError = error.userInfo[NSUnderlyingErrorKey] as? NSError {
+            event.message = SentryMessage(formatted: underlyingError.debugDescription)
+        } else {
+            event.message = SentryMessage(formatted: error.debugDescription)
+        }
```

Comment:
Code style suggestion (take it or leave it):
```suggestion
        let baseError: NSError = (error.userInfo[NSUnderlyingErrorKey] as? NSError) ?? error
        event.message = SentryMessage(formatted: baseError.debugDescription)
```